packages/xo-web/src/icons.scss

```diff
@@ -728,6 +728,11 @@
     @extend .fa-sign-out;
   }
 
+  &-file-text {
+    @extend .fa;
+    @extend .fa-file-text-o;
```

Comment:
IMHO `.fa-copy`

Reply:
I think a `.fa-file-text-o` is more representable.
In modal we write a text.
`Tooltip` explain that it is a copy-paste commands.

Reply:
@pdonias what do you think about this ?

Reply:
I think the icon needs to have horizontal lines to show that you can paste multiple lines. So I guess `.fa-file-text-o` is fine. But if you find a better icon with horizontal lines, I'm ok with that too, though.